plugins/sim_factory/SimFactory.cc

```diff
@@ -211,5 +211,5 @@ bool SimFactory::Load(const tinyxml2::XMLElement *_elem)
 
 
 
-  return false;
+  return true;
```

Comment:
This is a bit strange. It returns true no matter what?

Reply:
updated to variable to keep track of all factory spawn results, and that'll determine if load failed or not. [0ad7f43](https://github.com/gazebosim/gz-launch/pull/309/commits/0ad7f43784e210263ab925cd5465ab91ba484d2b)